app/azure/utils.ts

```diff
@@ -15,6 +15,20 @@ export function getSubscriptionResourceGroups(
   }));
 }
 
+export function aggregateResourceGroups(
```

Comment:
before I move a file to a different directory, I ask if it's really needed there. 

If we decide we need `aggregateResourceGroups`, then we need it only for the dashboard, so just put it in a file in the same directory as the dashboard

Reply:
The utils file is used in the createContainer view also which is why I moved it to the root. Are you suggesting here that I could have also put a utils file in the Dashboard directory if I had kept aggregateResourceGroups ?

Reply:
exactly